base/array.jl

```diff
@@ -1745,12 +1771,16 @@ julia> insert!(Any[1:6;], 3, "here")
 ```
 """
 function insert!(a::Array{T,1}, i::Integer, item) where T
+    @inline
```

Comment:
what is this inlining?

Reply:
This function - it's equivalent to an annotation on the definition (iiuc)